proto/src/determined/user/v1/user.proto

```diff
@@ -28,8 +28,8 @@ message User {
   // Bool denoting whether the user should be able to login with or change a
   // password.
   bool remote = 8;
-  // when the user last logged in
-  google.protobuf.Timestamp last_login = 9;
+  // when the user last authenticated
+  optional google.protobuf.Timestamp last_login = 9;
```

Comment:
I think `last_auth` is more accurate than `last_login`, but leave it as it is for now

Reply:
This hasn't been released yet right? 

I would try and avoid a backwards incompatible change to ```GetUsers``` from release to release?

Reply:
Not released yet.